src/utils/body.test.ts

```diff
@@ -232,18 +232,28 @@ describe('Parse Body Util', () => {
 
   it('should parse multiple values if key ends with `[]`', async () => {
```

Comment:
I think you don't need to change the name of the key. You can keep `file` as is.